deploy/setup.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+ENVCODE=$1
+LOCATION=$2
+PIPELINE_NAME=$3
+ENVTAG=$4
+
+
+set -x
```

Comment:
We have removed -e in the past to avoid abrupt exits.. I think let the debug mode be on and not merge any changes for that in this PR. We can take a separate disc for maintaining consistency on that

Reply:
Will capture this in backlog for future PRs.